test/integration/scheduler/preemption_test.go

```diff
@@ -295,7 +296,7 @@ var _ = ginkgo.Describe("Preemption", func() {
 					g.Expect(apimeta.FindStatusCondition(betaMidWl.Status.Conditions, kueue.WorkloadPreempted)).To(gomega.BeComparableTo(&metav1.Condition{
 						Type:    kueue.WorkloadPreempted,
 						Status:  metav1.ConditionTrue,
-						Reason:  "InCohort",
+						Reason:  preemption.InCohortReclamationReason,
```

Comment:
We are covering 2/3 cases, right? Could you add coverage for InCohortFairSharingReason please?